spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -104,7 +104,7 @@
             },
             "role": {"id": InvenioRdmRecordConverter::ROLE_OTHER}
           }],
-          "dates": [{"date": "2021-1-1", "type": "other", "description": "When the item was originally created."}],
+          "dates": [{"date": "2021-1-1", "type": {"id":"other"}, "description": "When the item was originally created."}],
```

Comment:
The date format should follow EDTF level 0 https://inveniordm.docs.cern.ch/reference/metadata/#dates-0-n which means double digit resolution i.e. it should be `"2021-01-01"`.

Reply:
Will the import still allow any dates? Or is it strictly EDTF level 0? This a free text field in DigitalHub, unfortunately.

Reply:
No, the import only allows EDTF level 0 dates.